tools/cldr-apps/js/src/views/NotificationPopup.vue

```diff
@@ -0,0 +1,51 @@
+<script setup>
+import { ref } from "vue";
+
+const message = ref("");
+const description = ref("");
+const formIsVisible = ref(false);
+
+/**
+ * Display a notification
+ *
+ * @param {String} mess the title message, displayed at the top
+ * @param {String} desc the more detailed description, which may be HTML
+ */
+function openWithMessageAndDescription(mess, desc) {
+  message.value = mess;
+  description.value = desc;
+  formIsVisible.value = true;
+}
+
+function onClose() {
+  formIsVisible.value = false;
+  message.value = description.value = "";
+}
+
+defineExpose({
+  openWithMessageAndDescription,
+});
+</script>
+
+<template>
+  <!-- The dialog layout and functionality are intended to resemble those of the dialogs
+   produced by cldrNotification.open, which is implemented with notification.open from Ant Design Vue.
+   One difference is that HTML is supported here for the description, which in general may come from
+   HTML messages generated by the Survey Tool back end.
+
+   References:
+     https://antdv.com/components/notification
+     https://antdv.com/components/modal
+  -->
+  <a-modal
+    v-model:visible="formIsVisible"
+    :title="message"
+    :footer="null"
+    :maskClosable="false"
+    :keyboard="false"
+    style="top: 24px; right: 24px; position: absolute"
+    @ok="onClose"
+  >
+    <div v-html="description" />
```

Comment:
🙈 

I'm OK with this for now, if we file a ticket (and add a TODO) to fix this as new tech debt…

Reply:
We have so many techdebt issues with "minor" priority already... but OK, I'll make another one :-)

I'm not sure how specific or general it should be. A general issue is that the back end often includes HTML in http responses. If I remember right, this v-html is an example of handling html that comes from an http response, though it's further complicated by cldrSurvey.testsToHtml adding further html...

I just made this ticket: https://unicode-org.atlassian.net/browse/CLDR-18013

Reply:
I'll follow up with adding a TODO referencing that ticket

Reply:
https://github.com/unicode-org/cldr/pull/4120